config/locales/en.yml

```diff
@@ -72,17 +72,23 @@ en:
         page_title: :thredded.nav.settings
       form:
         global_preferences_label: Global Settings
-        messageboard_notify_on_mention:
+        messageboard_auto_follow_topics:
           hint: >-
             When someone mentions you by your username (eg: @sam) in this messageboard you will follow the topic.
```

Comment:
Thredded also auto-follows the topics created by the user, and the topics the user replies to.

Since this setting only controls auto-follows on mention, it should probably be called `auto_follow_on_mention` throughout.